source/extensions/filters/http/grpc_json_transcoder/json_transcoder_filter.cc

```diff
@@ -887,6 +900,50 @@ bool JsonTranscoderFilter::maybeConvertGrpcStatus(Grpc::Status::GrpcStatus grpc_
   return true;
 }
 
+bool JsonTranscoderFilter::decoderBufferLimitReached(uint64_t buffer_length) {
+  if (!Runtime::runtimeFeatureEnabled(buffer_limits_runtime_feature)) {
+    return false;
+  }
+
+  if (buffer_length > decoder_callbacks_->decoderBufferLimit()) {
+    ENVOY_LOG(debug,
+              "Request rejected because the transcoder's internal buffer size exceeds the "
+              "configured limit: {} > {}",
+              buffer_length, decoder_callbacks_->decoderBufferLimit());
+    error_ = true;
+    decoder_callbacks_->sendLocalReply(
+        Http::Code::PayloadTooLarge,
+        "Request rejected because the transcoder's internal buffer size exceeds the configured "
+        "limit.",
+        nullptr, absl::nullopt,
+        absl::StrCat(RcDetails::get().GrpcTranscodeFailed, "_request_buffer_size_limit_reached"));
```

Comment:
```suggestion
        absl::StrCat(RcDetails::get().GrpcTranscodeFailed, "request_buffer_size_limit_reached"));
```

Same below

Reply:
The current commit results in the following string: `grpc_json_transcode_failure_request_buffer_size_limit_reached`

The suggested change would result in: `grpc_json_transcode_failurerequest_buffer_size_limit_reached`

That doesn't seem right. Keep in mind this is in `absl::StrCat`, the entire string is passed to `details` arg of `sendLocalReply`.

https://github.com/envoyproxy/envoy/blob/cfdd59a6280fed47ef4b881491c9ed0e96d4476d/include/envoy/http/filter.h#L403

Reply:
Ah OK I see, got it. Thanks.

Reply:
I think I also understand now why it was using braces before, sorry. I'm fine either way. Are you good with this or do you want to go back to braces?

Reply:
No worries. I prefer the braces for consistency. Reverted the last commit.